content/en/blog/2024/java-metric-systems-compared/index.md

```diff
@@ -285,13 +285,13 @@ The result is a configurable option unique to OpenTelemetry Java called
 what their memory mode is based on whether they read metric state concurrently
 or not. Right now you opt into the optimized memory behavior (which we call
 `MemoryMode.reusable_data`) via an
-[environment variable](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk-extensions/autoconfigure#exporters).
-In the future, the optimized memory mode will be enabled by default, since only
-exceptional cases need concurrent access to the metric state. It turns out that
-the objects holding the metric state (`MetricData` in OpenTelemetry Java terms)
-account for virtually all of the memory allocation in the collect cycle. By
-reusing these (along with other internal objects used to hold state), **we
-reduced the memory allocation of the core metric SDK by over 99%**. See
+[environment variable](/docs/languages/java/configuration/). In the future, the
```

Comment:
@jack-berg @open-telemetry/java-approvers is this change ok?

Reply:
Would be best to link to this section: https://opentelemetry.io/docs/languages/java/configuration/#properties-exporters